config.json

```diff
@@ -912,6 +912,16 @@
       "slug": "complex-numbers",
       "difficulty": 3,
       "topics": [ "Mathematics" ]
+    },
+    {
+      "uuid": "1af13ab1-04ea-5380-0958-32c65f0dbe41932275d",
+      "slug": "sublist",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
```

Comment:
Adding some topics from the [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) would be good.

Reply:
Will do!